llm/ext_server/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 set(TARGET ollama_llama_server)
 option(LLAMA_SERVER_VERBOSE "Build verbose logging option for Server" ON)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+if (DEFINED ENV{ASCEND_TOOLKIT_HOME})

```

Comment:
You can use LIBRARY_PATH env instead of changing CMkeLists.txt

Reply:
fixed